docs/_templates/versions.html

```diff
@@ -0,0 +1,29 @@
+{%- if current_version %}
+<div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+  <span class="rst-current-version" data-toggle="rst-current-version">
+    <span class="fa fa-book"> Other Versions</span>
+    v: {{ current_version.name }}
+    <span class="fa fa-caret-down"></span>
+  </span>
+  <div class="rst-other-versions">
+    <dl>
+      <dt>ROS Team Workspace - Galactic</dt>
```

Comment:
```suggestion
      <dt>ROS Team Workspace - Galactic and Rolling</dt>
```

Reply:
As long as the same image is used, symlinking it should be enough. However, i personally would strongly suggest to use different images for different versions of the docs. This allows users to quickly see what version they are using and avoids confusion.

Reply:
*github/workflows need to be updated.

Maybe it would also be useful to add a htmlproofer since the versioning in the sidebar is done by hand.